apps/examples/wc-vue3/tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "compilerOptions": {
     "allowJs": true, // allow JavaScript files to be imported
     "baseUrl": ".",
+    "ignoreDeprecations": "5.0",
```

Comment:
Out of interest – what does this do?

Reply:
Apparently we’re using a feature that’s going to be deprecated in TS 5.5. It did offer a solution, however it didn’t work when I tried it, so this was just a quick win to get the app building